src/TheThingsNetwork.cpp

```diff
@@ -287,7 +287,7 @@ uint8_t receivedPort(const char *s)
   return port;
 }
 
-TheThingsNetwork::TheThingsNetwork(Stream &modemStream, Stream &debugStream, ttn_fp_t fp, uint8_t sf, uint8_t fsb)
+TheThingsNetwork::TheThingsNetwork(SerialType &modemStream, Stream &debugStream, ttn_fp_t fp, uint8_t sf, uint8_t fsb)
```

Comment:
1. Random drive by comment but don't forget to update the [documentation](https://github.com/TheThingsNetwork/arduino-device-lib/blob/master/docs/TheThingsNetwork.md#class-thethingsnetwork) since the `Stream` is changing to a `SerialType`.
2. Why change the modemStream to a `SerialType` but not the debugStream?
3. Wouldn't it make sense to change the variable name from `modemStream` to `modemSerial` or something in that style to avoid confusion?